.gitlab-ci.yml

```diff
@@ -37,6 +37,7 @@ build-runner-image:
   tags: ["arch:amd64"]
   variables:
     PUSH_IMAGE: "false"
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
OMG did we always had it there ?? Or you added one ? I'm checking this with CI team and could not get a github token yet

Reply:
Just found https://docs.github.com/en/actions/security-guides/automatic-token-authentication thanks to @KevinFairise2

Reply:
This won't work, as the `GITHUB_TOKEN` is available only on Github Workflows :(